monkey/monkey_island/cc/services/node.py

```diff
@@ -62,7 +63,9 @@ def get_displayed_node_by_id(node_id, for_report=False):
 
     @staticmethod
     def get_node_label(node):
-        return node["os"]["version"] + " : " + node["ip_addresses"][0]
+        if node["domain_name"]:
+            node["domain_name"] = " ("+node["domain_name"]+")"
```

Comment:
What you're doing will change node["domain_name"] also outside context of this function.
What'll happen if you call get_node_label multiple times? you'll get multiple layers of parenthesis.
Use a local variable instead. If you're counting on this behavior on other functions, you'll have to change it.